src/main/java/org/opensearch/neuralsearch/processor/factory/NormalizationProcessorFactory.java

```diff
@@ -77,6 +78,13 @@ public SearchPhaseResultsProcessor create(
             Map<String, Object> combinationParams = readOptionalMap(NormalizationProcessor.TYPE, tag, combinationClause, PARAMETERS);
             scoreCombinationTechnique = scoreCombinationFactory.createCombination(combinationTechnique, combinationParams);
         }
+
+        TechniqueCompatibilityCheckDTO techniqueCompatibilityCheckDTO = TechniqueCompatibilityCheckDTO.builder()
```

Comment:
If you are not going to rename the method to "isCompatible" then your old class name ValidateNormalizationDTO 
 fits better

Reply:
Changed the method to `isTechniquesCompatible`